### ISSUE TITLE
HTTP 410 InvalidPartition each time emulator is restarted

### ISSUE THREAD

**Issue by lukepuplett** [Azure/azure-cosmos-dotnet-v2#745]
**Describe the bug**

Each time I restart the Cosmos emulator I get an error when I perform a query and I need to delete the database to resolve it and do my work.

The HTTP code is 410 and the message on the StorageException is "InvalidPartition".

**To Reproduce**

- Start the local Cosmos DB emu with Tables API enabled.
- Delete the Table database entirely.
- Use your app locally so it creates the DB again, etc. writes data to local Cosmos DB.
- All is good, app is working.
- Stop the emu, wait for it to close, then start it again as before.
- Use your app again and observe a long pause before a StorageException: InvalidPartition is thrown.

**Expected behavior**

Before yesterday, restarting the emulator was not a problem.

**Actual behavior**

Restarting the emu means having to delete my database and let my app code auto-generate it all again.

**Environment summary**

SDK Version: VS 16.3.6 and Cosmos Emulator 2.4.3.0 (50x45d91)
OS Version (e.g. Windows, Linux, MacOSX): Windows 10

**Additional context**

Yesterday, after rebooting to update Visual Studio to 16.3.6 I ran my app and received an HTTP 410 from the local Cosmos DB emulator over the Table API port.

I looked on Google and nothing, so I deleted the database in Storage Explorer and it recreated and all was okay again.

Today I reboot for a Windows Update and the same thing has happened.

    exception.Message = InvalidPartition
    exception.StackTrace =
       at Microsoft.Azure.Cosmos.Table.RestExecutor.TableCommand.Executor.<ExecuteAsync>d__1`1.MoveNext()
       at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
       at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
       at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
       at Evoq.Data.TableStorage.TableStorageHelper`1.<ExecuteOperationAsync>d__27.MoveNext()

**Comment by lukepuplett** [comment]
![Capture5](https://user-images.githubusercontent.com/5802524/67480445-d228fd00-f657-11e9-8220-27c08e0c947c.PNG)

Just a screenshot showing the partition does exist and my code is looking up a valid part and row key, I'm not going mad :)

**Comment by lukepuplett** [comment]
I did a Reset Data from the Systray menu and though in Windows Explorer it looked like it did nothing, when I restarted the emulator it give me a warning about the version of the emulator and the data in the folder; it wasn't giving me this before.

So to be sure, I stopped the emu and cleared the folder of everything using Windows Explorer and started the emu.

Everything is fine now. I can restart the emulator without having to delete the DB each time now. Closing.
